changeme.py

```diff
@@ -372,11 +371,20 @@ def render_creds(candidate, csrf):
 def check_http(req, session, candidate, config, sessionid=False, csrf=False):
     matches = list()
     data = None
+    headers = dict()
 
     url = get_base_url(req)
     logger.debug('[check_http] base url: %s' % url)
     urls = candidate['auth']['url']
-
+    if candidate['auth']['headers']:
```

Comment:
I only want to to build the headers if there's additional headers in the yml.  Wouldn't that return true no matter if those headers are present or not?  I'm looking for it to return false and move to the else where it just adds the useragent to the headers.